dashboard/app/views/home/_donor_footer.html.haml

```diff
@@ -9,8 +9,7 @@
     - else
       Your free education on Code.org is made possible by:
   - @homepage_data[:donorFooterOptions][:logos].each do |logo|
-    .logo
-      = image_tag asset_path("donor_logos/#{current_user.school_donor_name}/#{logo.split('/').last}")
+    = image_tag asset_path("donor_logos/#{current_user.school_donor_name}/#{logo.split('/').last}"), class: 'logo'
```

Comment:
Dropping the wrapper `div` and applying the logo class directly to the image fixed enforcing the max-height on the logo.